flink-ml-lib/src/test/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionTest.java

```diff
@@ -0,0 +1,279 @@
+
+        logisticRegression
+                .setFeaturesCol("test_features")
+                .setLabelCol("test_label")
+                .setWeightCol("test_weight")
+                .setMaxIter(1000)
+                .setTol(0.001)
+                .setLearningRate(0.5)
+                .setGlobalBatchSize(1000)
+                .setReg(0.1)
+                .setMultiClass("binomial")
+                .setPredictionCol("test_predictionCol")
+                .setRawPredictionCol("test_rawPredictionCol");
+        assertEquals(logisticRegression.getFeaturesCol(), "test_features");
+        assertEquals(logisticRegression.getLabelCol(), "test_label");
+        assertEquals(logisticRegression.getWeightCol(), "test_weight");
+        assertEquals(logisticRegression.getMaxIter(), 1000);
+        assertEquals(logisticRegression.getTol(), 0.001, TOLERANCE);
+        assertEquals(logisticRegression.getLearningRate(), 0.5, TOLERANCE);
+        assertEquals(logisticRegression.getGlobalBatchSize(), 1000);
+        assertEquals(logisticRegression.getReg(), 0.1, TOLERANCE);
+        assertEquals(logisticRegression.getMultiClass(), "binomial");
+        assertEquals(logisticRegression.getPredictionCol(), "test_predictionCol");
+        assertEquals(logisticRegression.getRawPredictionCol(), "test_rawPredictionCol");
+    }
+
+    @Test
+    public void testFeaturePredictionParam() {
+        Table tempTable = binomialDataTable.as("test_features", "test_label", "test_weight");
+        LogisticRegression logisticRegression =
+                new LogisticRegression()
+                        .setFeaturesCol("test_features")
+                        .setLabelCol("test_label")
+                        .setWeightCol("test_weight")
+                        .setPredictionCol("test_predictionCol")
+                        .setRawPredictionCol("test_rawPredictionCol");
+        Table output = logisticRegression.fit(binomialDataTable).transform(tempTable)[0];
+        assertEquals(
+                Arrays.asList(
+                        "test_features",
+                        "test_label",
+                        "test_weight",
+                        "test_predictionCol",
+                        "test_rawPredictionCol"),
+                output.getResolvedSchema().getColumnNames());
+    }
+
+    @Test
+    public void testFitAndPredict() throws Exception {
+        LogisticRegression logisticRegression = new LogisticRegression().setWeightCol("weight");
+        Table output = logisticRegression.fit(binomialDataTable).transform(binomialDataTable)[0];
+        verifyPredictionResult(
+                output,
+                logisticRegression.getFeaturesCol(),
+                logisticRegression.getPredictionCol(),
+                logisticRegression.getRawPredictionCol());
+    }
+
+    @Test
+    public void testSaveLoadAndPredict() throws Exception {
+        LogisticRegression logisticRegression = new LogisticRegression().setWeightCol("weight");
```

Comment:
Could we also save/load the estimator in this test, similar to `NaiveBayesTest::testSaveLoad`?